src/env-inl.h

```diff
@@ -445,7 +448,9 @@ inline std::vector<double>* Environment::destroy_ids_list() {
 }
 
 inline double Environment::new_async_id() {
-  return ++async_hooks()->uid_fields()[AsyncHooks::kAsyncUidCntr];
+  async_hooks()->uid_fields()[AsyncHooks::kAsyncUidCntr] =
+    async_hooks()->uid_fields()[AsyncHooks::kAsyncUidCntr] + 1;
+  return async_hooks()->uid_fields()[AsyncHooks::kAsyncUidCntr];
```

Comment:
is this only a style change, or does the previous operation not work anymore?

Reply:
The previous operation doesn’t work anymore, we’d need to overload `operator++` for that, and from there it quickly starts to devolved into supporting all integer operations